CMakeLists.txt

```diff
@@ -646,11 +651,22 @@ if(WITH_CAROTENE)
   endif()
 endif()
 
+if(HAVE_OPENVX)
```

Comment:
I believe this block should go a few lines above before carotene (order is important).